tests/e2e/support/objects/app-files/link/index.ts

```diff
@@ -79,4 +81,17 @@ export class Link {
     })
     await this.#page.goto(startUrl)
   }
+
+  async publicLinkVisible(linkName): Promise<string[]> {
+    const linkUrl = this.#linksEnvironment.getLink({ name: linkName })
+    const publicLink = await getPublicLinkVisibility({
+      page: this.#page,
+      linkName
+    })
+    return [linkUrl.url, publicLink]
+  }
+
+  async linkEditButtonVisibile(linkName): Promise<boolean> {
```

Comment:
same here, i would prefer a different naming like `buttonIsVisible` or similar... then it sounds more like a question which answers `yes` or `no`